scylla.go

```diff
@@ -71,6 +150,17 @@ func parseSupported(supported map[string][]string) scyllaSupported {
 	return si
 }
 
+func parseCQLProtocolExtensions(supported map[string][]string) []cqlProtocolExtension {
+	exts := []cqlProtocolExtension{}
+
+	lwtExt := newLwtAddMetaMarkExt(supported)
```

Comment:
You can check if the name is empty if you return the struct instead of a pointer.